submodules/TouchFreeUnityTooling/Runtime/Scripts/Connection/ConnectionManager.cs

```diff
@@ -84,13 +80,13 @@ public void Connect()
             //Read IP and port from config before we attempt to connect
             string ip = "127.0.0.1";
             string port = "9739";
-            string configPath = ConfigFileUtils.ConfigFileDirectory + "ServiceConfig.json";
-            if (File.Exists(configPath))
-            {
-                JObject obj = JObject.Parse(File.ReadAllText(configPath));
-                if (obj.ContainsKey("ServiceIP")) ip = obj["ServiceIP"].ToString();
-                if (obj.ContainsKey("ServicePort")) port = obj["ServicePort"].ToString();
-            }
+            // string configPath = ConfigFileUtils.ConfigFileDirectory + "ServiceConfig.json";
+            // if (File.Exists(configPath))
+            // {
+            //     JObject obj = JObject.Parse(File.ReadAllText(configPath));
+            //     if (obj.ContainsKey("ServiceIP")) ip = obj["ServiceIP"].ToString();
+            //     if (obj.ContainsKey("ServicePort")) port = obj["ServicePort"].ToString();
+            // }
```

Comment:
Commented it out deliberately because it was a hack AFAIK and introduces a dependency on newtonsoft json. @NatGreenwood-UL is it ok to leave this commented out or delete it?

Reply:
This was, yes, a hack, but more importantly it was a hack introduced _for the Unity Settings app_. We didn't ever intend for _Tooling_ to automatically connect itself to the Service on the device - that may actually be undesirable behaviour. 

The better option here is probably to introduce a way to configure the IP/Port in tooling, and then move this "check the Config File" logic to _only_ Settings app.

Reply:
Resolved this now. Both overlay and settings will read service config before any connect.